src/Resources/Users/Toybox.ts

```diff
@@ -20,4 +20,9 @@ export default class Toybox extends AbstractResource {
         method: 'PATCH',
         path: '/stock/{id}/add-to-toybox',
     });
+
+    public purchase: Function = this.createMethodFromPartialSpec({
```

Comment:
Ah, just noticed the type hinting for this function is wrong. If you look at the previous file in this PR (src/Resources/Users/Subscriptions.ts ) you can see what it should be.

Could you amend that on your original branch when you get a sec today please and PR to develop?

Reply:
done